components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -131,6 +132,10 @@ public class ServiceProvider implements Serializable {
     @XmlElement(name = TEMPLATE_ID)
     private String templateId;
 
+    @IgnoreNullElement
+    @XmlElement(name = TEMPLATE_VERSION)
+    private String templateVersion;
```

Comment:
Can you give an example of what type of values the `templateVersion` would hold?

Reply:
It can hold values like v1.0.0